src/rcc.rs

```diff
@@ -66,7 +66,7 @@ pub struct APB1 {
 }
 
 impl APB1 {
-    pub(crate) fn enr(&mut self) -> &rcc::APB1ENR {
```

Comment:
Why is this changed? :)

Reply:
Shouldn't be in this PR; meant it for a diff branch. Shouldn't be changed here.